python/llm/dev/benchmark/all-in-one/config.yaml

```diff
@@ -37,6 +37,7 @@ test_api:
   # - "deepspeed_transformer_int4_cpu"      # on Intel CPU, deepspeed autotp inference
   # - "transformers_int4_npu_win"           # on Intel NPU for Windows,  transformer-like API, (qtype=int4)
   # - "transformers_int4_loadlowbit_npu_win" # on Intel NPU for Windows, transformer-like API, (qtype=int4), use load_low_bit API. Please make sure you have used the save_npu.py to save the converted low bit model
+  - "transformers_openvino"                 # on Intel GPU, openvino API
```

Comment:
Before merging, we need to comment `"transformers_openvino"` and uncomment "transformer_int4_fp16_gpu"